core/src/main/java/com/datastax/oss/driver/internal/core/metadata/schema/queries/DefaultSchemaQueriesFactory.java

```diff
@@ -81,6 +82,19 @@ protected SchemaQueries newInstance(
     } else if (version.compareTo(Version.V4_0_0) < 0) {
       return new Cassandra3SchemaQueries(channel, refreshFuture, config, logPrefix);
     } else {
+
+      /* A bit of custom logic for DSE 6.0.x.  These versions report a Cassandra version of 4.0.0 but don't
+       * have support for system_virtual_schema tables supported by that version.  To compensate we return
+       * the Cassandra 3 schema queries here for those versions */
```

Comment:
Nit: could you avoid block-style comments (ie. use `//` style instead)?

Sometimes when you're on a test server and all you have is the command line and vi, it's convenient to use `/* ... */` to comment parts of the code, but that only works if it doesn't already contain such comments.

Reply:
Sure, although the block syntax is common even throughout the existing code base.  I personally prefer it (I find the double-slash syntax visually very distracting) but I don't care enough to go to the mat for it.